lib/etcdctl.py

```diff
@@ -178,7 +181,9 @@ def run(self, arguments, endpoints=None, api=3):
                 command.insert(2, endpoints)
 
         try:
-            return check_output(command, env=env).decode("utf-8")
+            return sp_run(
+                command, env=env, capture_output=True, check=True
+            ).stdout.decode("utf-8")
```

Comment:
why does this need to change?  i don't understand what's different.

Reply:
what's this for

Reply:
As far as I understand `check_output` doesn't capture `stderr`, and is technically a `subprocess.run` call according to the [pydocs](https://docs.python.org/3/library/subprocess.html#subprocess.check_output). This is so the exceptions could contain messages from `stderr`.

Reply:
Unfortunately, `capture_output` was added in py37, and we need to support bionic (py36).  If we need `stderr`, let's revert back to `check_output` and add a `stderr=subprocess.STDOUT` param and drop the subsequent `log(e.stderr, ERROR)` below.

There are a couple other ways to ensure stdout/stderr; if you prefer logging the streams separately, we can do that with sp_run or Popen. But given the previous impl, it's not clear that merging these with the `stderr=` arg will be any *worse* that what we currently do.